src/workerd/io/worker-interface.capnp

```diff
@@ -460,6 +462,10 @@ struct JsValue {
         }
       }
 
+      abortSignal :group {
+        signal @7 :AbortSignal;
```

Comment:
(Per my previous comment...)

This is sending an `rpc::AbortSignal` capability to the callee, which can be used to send an abort _back_ to the caller. But I think that's actually not what you want? The JavaScript `AbortSignal` type only lets you receive signals, not send them.

You seem to have correctly used StreamSink to go the other direction.

Reply:
Perhaps calling the RPC interface `AbortSignal` is creating confusion here, since the RPC interface actually more closely resembles `AbortController`?

Reply:
Yeah I think I will rename to AbortController